src/Microsoft.Data.SqlClient/netfx/src/Resources/Strings.Designer.cs

```diff
@@ -5550,6 +5550,15 @@ internal class Strings {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to When true, enables usage of the Asynchronous functionality in the .NET Framework Data Provider..
+        /// </summary>
+        internal static string DbConnectionString_AsynchronousProcessing {
+            get {
+                return ResourceManager.GetString("DbConnectionString_AsynchronousProcessing", resourceCulture);
+            }
+        }
```

Comment:
I could not see the usage of this newly added string and the other one SQL_AsyncConnectionRequired.

Reply:
Removed, and I don't quite understand why it was added. It feels like the base branch for this wasn't main or something like that.